client/404.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BlogIt</title>
+    <!-- add favicon -->
+    <link rel="stylesheet" href="/style.css" />
+  </head>
+  <body>
+    <div id="container">
+      <form action="" class="card">
+        <textarea
+          class="title"
+          name="title"
+          id="title"
+          placeholder="title"
+          rows="1"
+          required
+        ></textarea>
+        <input
+          type="text"
+          class="author"
+          name="author"
+          id="author"
+          placeholder="your name"
+        />
+        <textarea
+          name="content"
+          id="content"
+          class="content"
+          placeholder="blog"
+          required
+        ></textarea>
+        <button type="submit" class="button">PUBLISH</button>
```

Comment:
When you click publish, via copying the 404.html GET file:///C:/style.css net::ERR_FILE_NOT_FOUND pops up